integration/management-api-tests/src/test/java/org/wso2/micro/integrator/api/ManagementAPITest.java

```diff
@@ -0,0 +1,65 @@
+package org.wso2.micro.integrator.api;
+
+import org.apache.http.HttpResponse;
+import org.awaitility.Awaitility;
+import org.json.JSONObject;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeSuite;
+import org.wso2.esb.integration.common.utils.ESBIntegrationTest;
+import org.wso2.esb.integration.common.utils.clients.SimpleHttpClient;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+public abstract class ManagementAPITest extends ESBIntegrationTest {
+
+    protected static final String LIST = "list";
+    protected static final String COUNT = "count";
+    protected static String accessToken;
+    private static String endpoint;
+
+    @BeforeSuite
+    public void setEnvironment() throws Exception {
+        super.init();
+        accessToken = TokenUtil.getAccessToken(hostName, portOffset);
+        endpoint = "https://" + hostName + ":" + (DEFAULT_INTERNAL_API_HTTPS_PORT + portOffset) + "/management/";
+    }
+
+    @BeforeClass
+    protected void init() throws Exception {
+        super.init();
+    }
+
+    protected JSONObject sendHttpRequestAndGetPayload(String resourcePath, String accessToken) throws IOException {
```

Comment:
Do we need to pass the accessToken here? Can't we use the `accessToken` variable defined in this class?